bindings/python/src/pre_tokenizers.rs

```diff
@@ -229,7 +230,7 @@ macro_rules! setter {
 ///         Whether to add a space to the first word if there isn't already one. This
 ///         lets us treat `hello` exactly like `say hello`.
 #[pyclass(extends=PyPreTokenizer, module = "tokenizers.pre_tokenizers", name=ByteLevel)]
-#[text_signature = "(self, add_prefix_space=True)"]
+#[text_signature = "(self, add_prefix_space=True, trim_offsets=True, regex_type=\"original\")"]
```

Comment:
I think we should remove `trim_offsets`  since we're not actually parsing it nor feeding to ByteLevel object here (and it's not used by `pre_tokenizer` so it's actually better to not mention it here.